content.yaml

```diff
@@ -1015,3 +1015,127 @@ long_summary_md: |
 
   * [L. McCann: So You Think You Want to Run a Hackathon? Think Again.](https://medium.com/@elle_mccann/so-you-think-you-want-to-run-a-hackathon-think-again-f96cd7df246a)
   * [CivicPatterns.org: Build with, not for](https://civicpatterns.org/patterns/build-with-not-for/)
+  
+  
+---
+uuid:     nt-2021-01-11
+title:    Chaos im Radio am 11.01.2021
+subtitle: Der Chaostreff im Freien Radio Potsdam
+summary:  >
+  Die erste Folge des Jahres 2021. Wir reden über mobile Messenger, den (gefühlten) beinah-Zusammenbruch des europäischen Stromnetzes, 
+  den Congress und sind verwirrt wie immer.
+publicationDate: "2020-21-18T17:55:00+01:00"
+audio:
+  - url:      $media_base_url/2021-01-11-chaos-im-radio.mp3
```

Comment:
File name pattern changed.
I got you want to rename the show. But please stick to `YYYY_MM_DD-<Name>.mp3`

Reply:
hmpf, ist doch nur n Typo :-D

Reply:
Damn, in der Folge drüber hab ich ja auch schon `-` verwendet...